app/settings/forms.py

```diff
@@ -13,14 +19,29 @@ class PreferencesForm(FlaskForm):
     submit = SubmitField('Save Changes')
 
     def validate_username(self, username):
-        # Need to validate that username is not already used.
-        pass
+        """ Checks if the data has changed, if it has then run validation from app/auth/RegistrationForm"""
+        if current_user.username.lower() != username.data.lower():
+            print(f'current:{current_user.username} new: {username.data}')
```

Comment:
I assume this was for testing and no longer needs to be there.

Reply:
Removed.